src/Cache/Container/ActiveContainer.php

```diff
@@ -211,16 +211,20 @@ public function get(string $key, Transformation $transformation): string|int|arr
         return $this->buildFinalTransformation($transformation)->transform($unpacked_values);
     }
 
-    public function set(string $key, string|int|array|bool|null $value): void
+    public function set(string $key, string|int|array|bool|null $value, int $ttl = null): void
```

Comment:
The parameter MUST be `?int $ttl = null` since in PHP 8.4 this will be mandatory